scripts/rss/index.js

```diff
@@ -0,0 +1,26 @@
+import fetch from 'node-fetch'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'node-fetch'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->